payu/models/cesm_cmeps.py

```diff
@@ -190,6 +184,73 @@ def setup(self):
                 # TODO: copied this from other models. Surely we want to exit here or something
                 print('payu: error: Unable to find mod_def.ww3 file in input directory')
 
+    def _setup_checks(self):
+        # check pelayout fits within requested cpucount
+        cpucount = int(
+            self.expt.config.get('ncpus', multiprocessing.cpu_count())
+        )
+        all_realms = self.realms
```

Comment:
```suggestion
        all_realms = self.realms + ["glc", "lnd"]
```

Are `glc` and `lnd` necessary?

Reply:
No we can remove the `glc` and `lnd` sections from PELayout in our configs.